doc/api/fs.md

```diff
@@ -2194,7 +2194,7 @@ The following constants are meant for use with the [`fs.Stats`][] object's
 [MDN-Date]: https://developer.mozilla.org/en/JavaScript/Reference/Global_Objects/Date
 [Readable Stream]: stream.html#stream_class_stream_readable
 [Writable Stream]: stream.html#stream_class_stream_writable
-[inode]: http://www.linux.org/threads/intro-to-inodes.4130
+[inode]:https://en.wikipedia.org/wiki/Inode
```

Comment:
changed from http://www.linux.org/threads/intro-to-inodes.4130  to 
https://en.wikipedia.org/wiki/Inode